dashboard/config/locales/authored_hints.cs-CZ.yml

```diff
@@ -151,6 +191,17 @@ cs:
           než začne kreslit.
         2-3_Artist_Loops_New_9_b: Tohle V je ve skutečnosti prostě polovina čtverce.
           Jaký by byl kód k nakreslení půlky čtverce?
+      2-3 Artist Nested Loops 1_authored_hint:
+        2-3_Artist_Nested_Loops_1_a: Nezapomeňte vložit všechny šedé bloky dovnitř
+          vaší smyčky.
+      2-3 Artist Nested Loops 3_authored_hint:
+        2-3_Artist_Nested_Loops_3_a: "Potřebujete pomoc s úhly?  \n** 360/12 = 30
```

Comment:
the added whitespace between `**` and `360` here and below will break the formatting

Reply:
Are these things we should fix in this PR? Or just remove them and fix them in Crowdin?

Reply:
The most important thing is to fix them in Crowdin so they'll be good for the next sync. Depending on the severity of the issue, I also like to fix them manually in this PR; for that, it's generally fine to just update the synced out files directly.

Reply:
Fixed the ones in this file and the other you pointed out, both in the code and in Crowdin.